### ISSUE TITLE
Solve some HTML warnings

### ISSUE THREAD

**Issue by jfoclpf** [jfoclpf/autocosts#79]
Hi @mcristaldo , I know I'm being picky, but can you solve HTML these warnings?
https://validator.w3.org/nu/?doc=http%3A%2F%2Fautocosts.work%2FBR

I could give a try, but I don't want to touch `h1-h6` headings since you created them
Thank you

**Comment by mcristaldo** [comment]
Warning: https://www.dropbox.com/s/nx26lhibkp1kc2f/Captura%20de%20pantalla%202018-10-25%2020.04.41.png?dl=0

I'm not sure why it's showing this warning cause actually we have headings along the site, even in the specific line that the warning is pointing out we have headings, so not sure how to solve this.

I solved the other warning though.

Thanks

**Comment by jfoclpf** [comment]
Thanks for solving the other warning @mcristaldo 
Regarding that warning, I think it has to do h1-h6 hierarchies
Check this page:
https://www.w3.org/wiki/HTML/Usage/Headings/Missing

And we don't respect hiearchy in results.hbs, for example here
https://github.com/jfoclpf/autocosts/blob/master/src/views/partials/results.hbs#L5

What do you think?

**Comment by jfoclpf** [comment]
In [`main.hbs` in line 88](https://github.com/jfoclpf/autocosts/blob/master/src/views/main.hbs#L88), we have:
```
<section id="results">
    <div>
        {{> results }}
    </div>
    <div class="bottom-actions-bar">
        {{> bottom_actions_bar }}
    </div>
</section>
```

and the `{{> results }}` therein resolve into `results.hbs` which starts with `<section id="info-boxes">`.

Thus it is resolved as section into a section with no h1-g6 tags un-between, that is
```
<section>
  <section id="info-boxes">
  ...
  </section>
  <section id="avg-periodic-cost">
  ...
  </section>
   ...
</section>
```

with not `h1-h6` tags in between.

And the (html rules)[https://www.w3.org/wiki/HTML/Usage/Headings/Missing] about such warning say:

### Example code: simple document with an section that has a missing heading
```
<body>
 <h1>top level heading</h1>
  <section> <!-- empty section -> 
   <section><h3>(intended) 2nd level heading is calculated as a level 3 heading 
   due to being nested in an empty section </h3>  
   </section>
  </section>
```

**Comment by jfoclpf** [comment]
Solved with 078b3b7e9c351f4c04198e00481d442b785e2749, 4391e52d219e0944d2b32b7d48ffd1b9d05b4f40 and 650f8fe7a89c0c0eb54655935238cb72b4354be9
